src/PrestaShopBundle/Resources/views/Admin/Layout/login_layout.html.twig

```diff
@@ -105,14 +105,17 @@
               &copy; PrestaShop&#8482; 2007-{{ "now"|date("Y") }} - All rights reserved
             </a>
           </p>
-          <p class="text-center">
+          <p class="login-footer-social text-center">
             <a class="link-social link-twitter _blank" href="https://twitter.com/PrestaShop" title="Twitter">
+              <twig:ux:icon name="bi:twitter-x" height="16" width="16" aria-hidden="true"/>
               Twitter
```

Comment:
There was no text in the legacy page so you can remove those and only keep the icons

Reply:
ok done